### DISCUSSION TITLE
Regarding Github Actions, do instances persist? (i.e MySQL)

### DISCUSSION THREAD

**Discussion by TheAxZim** [community/community#26129] [Actions]
<p>Hello!</p>
<p>I’ve not yet tried to do this, but after taking a look at Github Actions, I wanted to mimic what we do currently in CircleCI.</p>
<p>I want to start a docker image (MySQL) and then run my application, which will run queries and tests on the MySQL instance. On CircleCI though, we also have to wait for our MySQL instance to be ready and willing to accept connections to the database before we can run our tests.</p>
<p>Is this possible on Actions? Could I create an action to launch a MySQL instance and then refer back to it?</p>
<p>Thanks and sorry if I misunderstood anything!</p>
<p><strong>Edit:</strong> if someone could provide an example… that would be most amazing!!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/theaxzim">@theaxzim</a>,</p>
<p>Thanks a lot for being here! Unfortunately, It wouldn’t persist across Actions. You can create an external database using something like Heroku and use that though. Their free tier should be generous enough. I hope that helps!</p>
